packages/logs/src/transport/startLogsBridge.ts

```diff
@@ -8,6 +8,6 @@ export function startLogsBridge(lifeCycle: LifeCycle) {
   const bridge = getEventBridge<'log', LogsEvent>()!
 
   lifeCycle.subscribe(LifeCycleEventType.LOG_COLLECTED, (serverLogsEvent: LogsEvent & Context) => {
-    bridge.send('log', serverLogsEvent)
+    bridge.send('log', serverLogsEvent, serverLogsEvent.view.id)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏do we need to pass view id when the event is not a record?

Reply:
If we don't need it, let's not do it.